scripts/process_spraggins.sh

```diff
@@ -13,7 +13,8 @@ main() {
     ZARR_OUT="$OUTPUT/spraggins.ims.zarr"
     JSON_OUT="$OUTPUT/spraggins.ims.json"
 
-    CLI_ARGS="--imzml_file $IMZML_IN --ibd_file $IBD_IN --ims_zarr $ZARR_OUT --ims_metadata $JSON_OUT"
+    RELEASE=${CLOUD_TARGET//vitessce-data\//}
```

Comment:
Why the backslash?

Reply:
```
echo ${CLOUD_TARGET//vitessce-data//}
//0.0.24/master_release
```

```
echo ${CLOUD_TARGET//vitessce-data\//}
0.0.24/master_release
```

Reply:
Cool: I wasn't remembering the replacement syntax. Sorry!